com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/CollectionBuilder.java

```diff
@@ -252,6 +252,16 @@ private void countAllComponentTypes(Project project, CollectionDesc.Builder buil
         }
     }
 
+    private void createResourcePropertyTasks(List<ComponentPropertyDesc> overrideProps, IResource input) throws CompileExceptionError {
+        for (ComponentPropertyDesc compProp : overrideProps) {
+            Collection<String> resources = PropertiesUtil.getPropertyDescResources(project, compProp.getPropertiesList());
+            for(String r : resources) {
+                IResource resource = BuilderUtil.checkResource(project, input, "resource", r);
+                PropertiesUtil.createResourcePropertyTextureTask(project, resource, input);
```

Comment:
I think it's too confusing to call "create texture task" on _all_ resources.
Also, what about other resource types? Shouldn't those be created too?

I suggest you create a "createResourcePropertyTask", and inside _that_ one, check the type of the resource.

Reply:
Yes, it makes sense, I'll fix that. Thank you